### PR TITLE
Add send letter prod

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-868

### Change description ###

add send letter to prod aks

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/common/rpe/send-letter-service.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rpe-send-letter-service
+    version: 0.0.9
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rpe/send-letter-service:latest
+      environment:
+        SMTP_HOST: smtp.office365.com
```
Comment (timja) [inline k8s/prod/common/rpe/send-letter-service.yaml]: should probably be in the defaults?

File: k8s/prod/common/rpe/send-letter-service.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        FTP_DOWNTIME_FROM: 16:00
+        FTP_DOWNTIME_TO: 17:00
+        UPLOAD_SUMMARY_REPORT_ENABLED: true
+        ENCRYPTION_ENABLED: true
+    global:
+      environment: aat
```
Comment (timja) [inline k8s/prod/common/rpe/send-letter-service.yaml]: :shipit: 
```suggestion
      environment: prod
```

File: k8s/prod/common/rpe/send-letter-service.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rpe/send-letter-service:latest
+      environment:
+        SMTP_HOST: smtp.office365.com
+        FTP_DOWNTIME_FROM: 16:00
+        FTP_DOWNTIME_TO: 17:00
+        UPLOAD_SUMMARY_REPORT_ENABLED: true
+        ENCRYPTION_ENABLED: true
```
Comment (nitinprabhu) [inline k8s/prod/common/rpe/send-letter-service.yaml]: is it not enabled by default now ?

Comment (mdayican) [reply k8s/prod/common/rpe/send-letter-service.yaml]: default is true as well ,just in case  added for production

Comment (nitinprabhu) [reply k8s/prod/common/rpe/send-letter-service.yaml]: Please remove it if it default already

Comment (mdayican) [reply k8s/prod/common/rpe/send-letter-service.yaml]: removed

**Pull request merged**